internal/console/termios/termios_windows.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// Copyright 2019 The Go Authors. All rights reserved.
```

Comment:
The code in this file has been adapted from https://github.com/golang/term/blob/master/term_windows.go